call-of-papers/src/pages/Lectures/LectureForm.js

```diff
@@ -201,54 +222,55 @@ const LectureForm = () => {
 
             <Form.Item
               label="Minibiografia:"
-              name="miniBio"
             >
               <TextArea
                 rows={4}
-                {...formik.getFieldProps("miniBio")}
+                name="miniBio"
+                onChange={formik.handleChange} value={formik.values.profile.apresentation}
```

Comment:
Não seria melhor ter extraído as propriedades antes?
`const { apresentation, linkedinLink, facebookLink, twitterLink} = formik.values.profile`

Reply:
La, subi outro commit com uma alteração parecida com essa. Funcionou até certo ponto. Se quiser dar uma olhada... eu apaguei aquele Object.assign() que usava para "forçar" os eventos nos radios, checkboxes e select options e eles pararam de funcionar. Se eu coloco isso de volta, a aplicação entra em um loop infinito.